src/angular/styles/_typography.scss

```diff
@@ -216,6 +246,7 @@
 
   // Shadow
   --sbb-box-shadow-width: calc(var(--sbb-border-width) * 4);
+  --sbb-box-shadow-after-width: calc(var(--sbb-border-width) * 2);
```

Comment:
This should no be a var in global scope. In the datepicker usage it can be any size, the real width is determined by clip path. We can als use --sbb-box-shadow-width or just no var ;)